python/README.md

```diff
@@ -0,0 +1,11 @@
+# Python Bindings
+
+This directory contains experimental Python bindings to a small subset of Nix's functionality. These bindings are very fast since they link to the necessary dynamic libraries directly, without having to call the Nix CLI for every operation.
+
+Originally these bindings were created by [@Mic92](https://github.com/Mic92) and called [Pythonix](https://github.com/Mic92/pythonix). However, after Nix changed the internal library interface multiple times, therefore requiring Pythonix to be updated, it wasn't worth the third-party time investment anymore and the Pythonix repository was archived. At that point it only worked for Nix versions below 2.4. By having these bindings upstream, they can be updated right along the C++ code that defines the library interface.
```

Comment:
I think it's nice to credit the original author, but we don't have to elaborate as much on the history.

Also suggesting a short paragraph about it being experimental.

```suggestion
Thanks to [@Mic92](https://github.com/Mic92) who wrote [Pythonix](https://github.com/Mic92/pythonix) which these bindings were originally based on, before they became the official bindings that are part of the Nix project.

Note that the Python bindings are new and experimental. The interface is likely to change based on known issues and user feedback.
```

Reply:
I find this context quite useful to understand the project's motivation and why it's in nixpkgs, so I'd be in favor of leaving it in.